<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransRotateZoom</class>
 <widget class="QDialog" name="TransRotateZoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>旋转/缩放</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>旋转/翻转</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSlider" name="horizontalSlider_rotate">
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_rotate">
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QSlider" name="horizontalSlider_per_rotate">
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_per_rotate">
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QSlider" name="horizontalSlider_perspective">
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBox_perspective">
          <property name="minimumSize">
           <size>
            <width>52</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>52</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="minimum">
           <number>0</number>
          </property>
          <property name="maximum">
           <number>90</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_reset_rotate">
          <property name="text">
           <string>重置旋转</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>偏移</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>水平：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_offset_x">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_offset_x">
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <double>-1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>垂直：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="horizontalSlider_offset_y">
          <property name="minimum">
           <number>-100</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_offset_y">
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <double>-1.000000000000000</double>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_reset_offset">
          <property name="text">
           <string>重置偏移</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>缩放</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QSlider" name="horizontalSlider_zoom">
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_zoom">
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="maximum">
           <double>10.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_reset_zoom">
          <property name="text">
           <string>重置缩放</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QPushButton" name="pushButton_reset">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_rotate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_rotate</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_rotate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_rotate</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_per_rotate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_per_rotate</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_per_rotate</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_per_rotate</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_perspective</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_perspective</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_perspective</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_perspective</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>TransRotateZoom</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>TransRotateZoom</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
